cluster/charts/rook/templates/psp.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.pspEnable }}
+# PSP for rook-operator
+# name starts with 00- helps in policy order to keep it at the top - https://kubernetes.io/docs/concepts/policy/pod-security-policy/#policy-order
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: 00-rook-operator
+spec:
```

Comment:
can we add rook to the `allowedFlexVolumes`?

Reply:
`allowedFlexVolumes` is not supported in 1.6, it broke the build before.

Reply:
btw, #1644 will be merged soon to remove support for 1.6, so the `allowedFlexVolumes` could be added soon

Reply:
ok..cool